VideoLocker/res/values/strings.xml

```diff
@@ -442,11 +442,10 @@
     <!--  Email subject line when user attempts to submit feedback about the app -->
     <string name="email_subject">Customer Feedback</string>
 
-    <!-- Message indicating videos deleted. Argument is number of videos deleted -->
-    <plurals name="deleted_videos">
-        <item quantity="one">%s video deleted</item>
-        <item quantity="other">%s videos deleted</item>
-    </plurals>
+    <!-- Message indicating one video deleted-->
+    <string name="deleted_video">{video_num} video deleted</string>
```

Comment:
So you would do: String formatString = getResources().getQuantityString(R.plurals.deleted_videos, deletedVideoCount)

Phrase.from(formatString).put(key, value)